tests/genomic_file/test_genomic_file_resources.py

```diff
@@ -18,7 +18,7 @@
 
 GENOMICFILE_URL = 'api.genomic_files'
 GENOMICFILE_LIST_URL = 'api.genomic_files_list'
-EXPECTED_TOTAL = ENTITY_TOTAL + 102*2
+EXPECTED_TOTAL = ENTITY_TOTAL + 1*2
```

Comment:
I think we were using >100 gfs so we could run some pagination tests. Could we make it so that more gfs are put in to the test data?